rotkehlchen/chain/evm/decoding/structures.py

```diff
@@ -67,8 +68,11 @@ class TransferEnrichmentOutput(NamedTuple):
     matched_counterparty is optionally set if needed for enrichment rules
     and is used in post-decoding rules like in the case of balancer
     """
+    enriched: bool = False
```

Comment:
This argument makes little sense. I already wrote about it in discord: https://discord.com/channels/657906918408585217/1080783409640648765/1104155914367418429

If the only reason for the existence of this is to save a few characters when typing undo this change.

> so instead of having to do always TransferEnrichmentOutput(matched_counterparty=MY_COUNTERPARTY) we return now SUCCESSFUL_ENRICHMENT_OUTPUT

This is not a good reason to add a whole new argument to a context that is passed around in memory across the entrie decoder stack, multiplied by an ever increasing number of decoders. This should be kept as lean as possible.